src/Apps/Sell/Components/BottomFormNavigation.tsx

```diff
@@ -102,8 +110,9 @@ const BottomFormNextButton = () => {
     <Button
       variant="primaryBlack"
       disabled={!isValid}
-      loading={isSubmitting}
+      loading={isSubmitting || loading}
```

Comment:
I've just checked, and I think we can remove this additional loading state and derive the state directly from Formik. I think this would make things way simpler because we don't have another (kind of generic) state within the context.

The following code could either be here in the BottomFormNavigation or live in the context.

What do you think?
```
  const { isValid, submitForm, values } = useFormikContext()

  const isPhotoUploading = (values as PhotosFormValues)?.photos?.some(
    photo => photo.loading
  )
  
  ...
  loading={isSubmitting || isPhotoUploading}
  ...
  ```

Reply:
I am not strongly opinionated, but this was my logic: I didn't want to put a **Photo step**-related code in a generic component, that's why I pass `loading` property from Photo component itself. Imagine we have another component like that - we will need to add more and more checks to the `BottomFormNavigation` component. 

@olerichter00

Reply:
Let's go with your solution and keep the state in the context 👍